app/views/content/non-uk-teachers/international-qualified-teacher-status.md

```diff
@@ -20,49 +20,45 @@ keywords:
     - Qualified teacher status
 ---
 
-International qualified teacher status (iQTS) is a teaching qualification backed by the UK government. iQTS meets the same high standards as [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts), and leads to the automatic award of QTS. You can train where you live and work, with no need to visit England.
+International qualified teacher status (iQTS) is a teaching qualification that's the equivalent of [English qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts). iQTS enables you to train to teach in the country where you live and work, with no need to visit England.
 
 ## Why choose iQTS?
 
-iQTS is offered by world-class English teacher training providers working with schools in countries around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your region, and get in-person support from a trained mentor.
+iQTS is offered by teacher training providers in England that work with schools around the world. Alongside online tutorials and lectures, you’ll spend a minimum of 120 days gaining practical experience in a school in your country. You'll also get in-person support from a trained mentor.
 
-iQTS allows you to train to teach children aged 5 to 11, or children aged 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
+iQTS allows you to train to teach pupils aged 5 to 11, or pupils 11 to 18 in the full range of [secondary school subjects taught in English schools](https://www.gov.uk/national-curriculum).
 
-When you complete the course, you’ll have the skills, experience and knowledge to become an excellent teacher. You will also automatically be awarded English qualified teacher status (QTS) with no further fees or training. QTS is recognised all over the world as a teaching gold standard and will also qualify you to teach in England.
+When you complete the course, you’ll have the skills, experience and knowledge to become an excellent teacher. 
 
 If you decide to [teach in England](/non-uk-teachers/teach-in-england-if-you-trained-overseas), as an early career teacher you’ll benefit from a two-year induction period alongside your work in a school. You’ll earn a salary and get mentoring and continuous professional development to help you succeed as a teacher.
```

Comment:
This content is repeated further down the page in the `Early career teacher induction` section (as Emma points out) - can it be removed here?